drivers/hmis_external_apis/app/jobs/hmis_external_apis/publish_external_forms_job.rb

```diff
@@ -9,6 +9,7 @@
 #
 # currently this needs to be run manually
 class HmisExternalApis::PublishExternalFormsJob
```

Comment:
@gigxz should this inherit from BaseJob?

Reply:
could be. @ttoomey was it intentional to make this and `ConsumeExternalFormSubmissionsJob` _not_ inherit from BaseJob?